jena-arq/src/main/java/org/apache/jena/riot/system/ParserProfile.java

```diff
@@ -49,6 +49,12 @@
     /** Create a triple */
     public Triple createTriple(Node subject, Node predicate, Node object, long line, long col);
 
+    /** Check a triple */
```

Comment:
Might be nice to include a Javadoc note declaring what to do for bad tuples. Presumably `throw` something, but is there a specific type to throw that would be best?